README.md

```diff
@@ -30,12 +31,15 @@ script:
   - go test
 ignore:
   - .git
+  - myfile.ext
 verbose: true
 ```
 
-will make snag run the commands `echo "hello world"` and `go test`,
-ignoring changes in the `.git` directory, and returning output on success
-through the `verbose` option.
+By default, snag will watch all files/folders within the current directory recurssively.
+The ignore section in the yaml file will tell snag to ignore any changes that happen
```

Comment:
We can get away with not saying `yaml file` since it's already specified up there.